src/schema/objects/columns.yaml

```diff
@@ -162,6 +162,34 @@ duration:
   type: number
   unit: s
   minimum: 0
+x_coordinate:
+  name: x_coordinate
+  display_name: Gaze x-coordinate
+  description: |
+    Gaze position x-coordinate of the recorded eye, in the coordinate
+    units specified in the corresponding metadata sidecar.
+  type: number
+y_coordinate:
+  name: y_coordinate
+  display_name: Gaze y-coordinate
+  description: |
+    Gaze position y-coordinate of the recorded eye, in the coordinate
+    units specified in the corresponding metadata sidecar.
+  type: number
+pupil_size:
+  name: pupil_size
+  display_name: Pupil size
+  description: |
+    Pupil size or area of the recorded eye, in the
```

Comment:
Are we recording the particular type of pupil size that's being recorded here ('diameter' v/s 'area')?

The data that comes out of Eyelink is always in 'arbitrary units' ([Eyelink documentation](https://natmeg.se/onewebmedia/EL1000_UserManual_1.52.pdf): page 106, 4.4.3) and the actual data type is stored in the settings  ([Eyelink documentation](https://natmeg.se/onewebmedia/EL1000_UserManual_1.52.pdf): page 25 'Pupil Size Data').

Reply:
This could be another settings, do you feel like adding another entry to the table called something like pupil_type ?